### PR TITLE
Release 0.0.22

### PR DESCRIPTION
**Pull request by vontell**

### CODE REVIEW

File: docs/changelog.mdx [hunk 1]
```diff
@@ -4,6 +4,33 @@ sidebar_label: 'Changelog'
...
+_June 18, 2024_
+
+**New Features**
+
+- The SDK now supports recording games using the legacy input system and Unity UI.
+- New data collection format for improved state display within the scenario builder and validation results page.
+- New Bot Segment data format for stitching together various bot types, such as monkey testing bots combined with recordings.
+- Bounding boxes show entity locations within the gameplay session for improved debugging.
+
+## Release 0.0.21
```
Comment (vontell) [inline docs/changelog.mdx:18]: I guess I never made a release note for the last release, so added that here as well.

Comment (RG-nAmKcAz) [reply docs/changelog.mdx:18]: I think we need to differentiate ClientUI vs SDK in these lines

Comment (vontell) [reply docs/changelog.mdx:18]: Adjusted this slightly, but wasn't 100% sure which part was the unclear part - let me know if this is better!

Comment (abeizer) [reply docs/changelog.mdx:18]: I think the "bounding boxes" part needs to specify that it's a client UI feature

**Review by abeizer — APPROVED** [review]

**Review by svolokh — APPROVED** [review]

**Review by addisonbgross — APPROVED** [review]

**Pull request merged**